dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -106,10 +106,10 @@ def cleanLength(length):
             raise ValueError("Length must be positive")
         return length
 
-    @staticmethod
-    def cleanRow(row, fileType, validationInterface):
+    @classmethod
+    def cleanRow(cls, row, fileType, interfaces):
         for key in row.keys():
-            field_type = validationInterface.getColumn(key, fileType).field_type.name
+            field_type = interfaces.validationDb.getColumn(key, fileType).field_type.name
```

Comment:
When I wrote the padField function I wrote it to take the full interfaceHolder object since that's what we typically pass around.  Then when I added it to the cleanRow function I needed the interfaceHolder object available to pass into it, so I made this change.  I'll change padField to just require the validation interface and put this back the way it was.